larq/metrics.py

```diff
@@ -0,0 +1,99 @@
+import tensorflow as tf
+from larq import utils
+import numpy as np
+
+
+class MeanChangedValues(tf.keras.metrics.Mean):
```

Comment:
I didn't expose the metric in the public API and docs for now, so we can change it if we need to. I would like to make this metric lazy and not require users to pass in a static shape during construction, but this needs more thinking to prevent unnecessary nodes from being added to the graph.